nss-cohort-40/banagzon-workforce-mgt-upper-class-twits
# Adi review

# Description
Created the Department form and have it posting to the database. Redirects to departments after.
Fixes # (issue)
## Type of change
Please delete options that are not relevant.
# Testing Instructions
Check into branch adi_review and run the server. Navigate to departments/form and enter a new department
# Checklist:
